### DISCUSSION TITLE
Teams has no window decorations

### DISCUSSION THREAD

**Discussion by harridu** [fvwmorg/fvwm3#617] [Q&A]
MS Teams doesn't have Window decorations, and it behaves weird wrt placement on the desktop, stealing focus, moving to other screens, etc. What are your recommendations about setting Styles for Teams?

**Comment by ThomasAdam** [comment]
Swallow it inside FvwmButtons. It's what I do.

**Comment by harridu** [comment]
You might want to consider [teams-for-linux](https://github.com/IsmaelMartinez/teams-for-linux). I am using it for quite some time now.

**Comment by harridu** [comment]
```
Style   teams-for-linux !MWMDecor
```
did the trick.
